css/style.css

```diff
@@ -1 +1,6 @@
-
+html,
+body {
+	font-size: 15px;
+	font-family: 'montserrat';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6087879)